addons/account/models/product.py

```diff
@@ -118,6 +118,22 @@ def _check_uom_not_in_invoice(self):
                 "If you want to change its Unit of Measure, please archive this product and create a new one."
             ))
 
+    @api.model
+    def _force_default_tax(self, products, companies):
+        if not products:
```

Comment:
This shouldn't happen if we do a `create([])` for instance

Reply:
not sure I got your comment right... I changed the call in `create()`